nodejs.org/locale/uk/foundation/tsc/index.md

```diff
@@ -32,10 +35,12 @@ can be found on the project's [GitHub profile](https://github.com/orgs/nodejs/pe
 * James M Snell ([jasnell](https://github.com/jasnell))
 * Jeremiah Senkpiel ([Fishrock123](https://github.com/Fishrock123))
 * Julien Gilli ([misterdjules](https://github.com/misterdjules))
+* Michael Dawson ([mhdawson](https://github.com/mhdawson))
 * Rod Vagg ([rvagg](https://github.com/rvagg))
 * Shigeki Ohtsu ([shigeki](https://github.com/shigeki))
+* Steven R Loomis ([srl295](https://github.com/srl295))
```

Comment:
I think this is probably out of date, those members were removed when IBM acquired StrongLoop. Official list is at https://github.com/nodejs/TSC#current-members (The English website may be out of date.)